### ISSUE TITLE
Does ebs csi driver support Windows Server 2022?

### ISSUE THREAD

**Issue by KlwntSingh** [kubernetes-sigs/aws-ebs-csi-driver#1405]
/triage support

**Comment by torredil** [comment]
Currently, the following Windows OS versions are supported:
```
ltsc2019: mcr.microsoft.com/windows/servercore:ltsc2019
20H2: mcr.microsoft.com/windows/servercore:20H2
1809: mcr.microsoft.com/windows/servercore:1809
```
We should consider building for `ltsc2022` as well https://github.com/aws/containers-roadmap/issues/1710.

**Comment by KlwntSingh** [comment]
Thank you for your answer!

**Comment by ConnorJC3** [comment]
Update: the most recently released driver version (1.12.0) should now be compatible with 2022 LTSC hosts.
